diff -uraN peazip-10.0.0.src/dev/project_peach.lpi omv-peazip-10.0.0.src/dev/project_peach.lpi
--- peazip-10.0.0.src/dev/project_peach.lpi	2024-10-19 15:00:12.000000000 +0200
+++ omv-peazip-10.0.0.src/dev/project_peach.lpi	2024-10-26 17:51:28.907226327 +0200
@@ -29,21 +29,18 @@
         <Mode0 Name="default"/>
       </Modes>
     </RunParams>
-    <RequiredPackages Count="4">
+    <RequiredPackages Count="3">
       <Item1>
-        <PackageName Value="MetaDarkStyle"/>
-      </Item1>
-      <Item2>
         <PackageName Value="SynEdit"/>
         <MinVersion Major="1" Valid="True"/>
+      </Item1>
+      <Item2>
+        <PackageName Value="LCL"/>
       </Item2>
       <Item3>
-        <PackageName Value="LCL"/>
-      </Item3>
-      <Item4>
         <PackageName Value="FCL"/>
         <MinVersion Major="1" Valid="True"/>
-      </Item4>
+      </Item3>
     </RequiredPackages>
     <Units Count="202">
       <Unit0>
diff -uraN peazip-10.0.0.src/dev/project_pea.lpi omv-peazip-10.0.0.src/dev/project_pea.lpi
--- peazip-10.0.0.src/dev/project_pea.lpi	2024-10-02 15:54:48.000000000 +0200
+++ omv-peazip-10.0.0.src/dev/project_pea.lpi	2024-10-26 17:46:26.313127173 +0200
@@ -28,13 +28,9 @@
         <Mode0 Name="default"/>
       </Modes>
     </RunParams>
-    <RequiredPackages Count="2">
-      <Item1>
-        <PackageName Value="MetaDarkStyle"/>
-      </Item1>
-      <Item2>
+    <RequiredPackages Count="1">
         <PackageName Value="LCL"/>
-      </Item2>
+      </Item1>
     </RequiredPackages>
     <Units Count="55">
       <Unit0>
