diff -up peazip-9.9.0.src/dev/project_pea.lpi.1~ peazip-9.9.0.src/dev/project_pea.lpi
--- peazip-9.9.0.src/dev/project_pea.lpi.1~	2024-08-03 19:51:56.649950823 +0200
+++ peazip-9.9.0.src/dev/project_pea.lpi	2024-08-03 19:52:28.134376832 +0200
@@ -28,13 +28,10 @@
         <Mode0 Name="default"/>
       </Modes>
     </RunParams>
-    <RequiredPackages Count="2">
+    <RequiredPackages Count="1">
       <Item1>
-        <PackageName Value="MetaDarkStyle"/>
-      </Item1>
-      <Item2>
         <PackageName Value="LCL"/>
-      </Item2>
+      </Item1>
     </RequiredPackages>
     <Units Count="53">
       <Unit0>
diff -up peazip-9.9.0.src/dev/project_peach.lpi.1~ peazip-9.9.0.src/dev/project_peach.lpi
--- peazip-9.9.0.src/dev/project_peach.lpi.1~	2024-08-03 19:51:56.649950823 +0200
+++ peazip-9.9.0.src/dev/project_peach.lpi	2024-08-03 19:52:56.902801434 +0200
@@ -29,21 +29,18 @@
         <Mode0 Name="default"/>
       </Modes>
     </RunParams>
-    <RequiredPackages Count="4">
+    <RequiredPackages Count="3">
       <Item1>
-        <PackageName Value="MetaDarkStyle"/>
-      </Item1>
-      <Item2>
         <PackageName Value="SynEdit"/>
         <MinVersion Major="1" Valid="True"/>
+      </Item1>
+      <Item2>
+        <PackageName Value="LCL"/>
       </Item2>
       <Item3>
-        <PackageName Value="LCL"/>
-      </Item3>
-      <Item4>
         <PackageName Value="FCL"/>
         <MinVersion Major="1" Valid="True"/>
-      </Item4>
+      </Item3>
     </RequiredPackages>
     <Units Count="201">
       <Unit0>
