diff -uraN peazip-7.7.0.src/rmd160.pas omv-peazip-7.7.0.src/rmd160.pas
--- peazip-7.7.0.src/rmd160.pas	2020-09-02 17:38:18.000000000 +0200
+++ omv-peazip-7.7.0.src/rmd160.pas	2021-01-16 15:29:27.274050541 +0100
@@ -125,7 +125,7 @@
 
 
 {$ifdef FPC}
-  {$ifndef CPUARM}
+  {$if not(defined(CPUARM)) and not(defined(CPUAARCH64))}
     {$asmmode intel}
   {$endif}
 {$endif}
diff -uraN peazip-7.7.0.src/tsc.pas omv-peazip-7.7.0.src/tsc.pas
--- peazip-7.7.0.src/tsc.pas	2017-11-10 17:59:40.000000000 +0100
+++ omv-peazip-7.7.0.src/tsc.pas	2021-01-16 15:30:10.811210052 +0100
@@ -95,7 +95,7 @@
 
 implementation
 
-{$ifdef CPUARM}
+{$if defined(CPUARM) or defined(CPUAARCH64)}
 
 {$ifdef WINCE}
 
